src/general/StringUtils.cs

```diff
@@ -41,4 +41,9 @@ public static string FormatNumber(this double number, bool withSuffix = true)
 
         return number.ToString("0.#", CultureInfo.CurrentCulture);
     }
+
+    public static string FormatDecimal(this double number)
```

Comment:
Isn't this specifically for formatting with just one decimal place? I think the name should reflect that.

Reply:
Went ahead and made the function more generic.